.github/workflows/auto-create-milestone.yaml

```diff
@@ -8,14 +8,14 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: run
-        uses: ./
+      - uses: ./
+        id: run
```

Comment:
Is `id` right here? Shouldn't this be `name`?

Reply:
```
{{ steps.run.outputs.milestones }}
```

`id` is works.
`name` is works?

Reply:
I don't follow. Why the change from `name` to `id`?

Reply:
@erunion 
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-output-as-url

please read this.
name don't work

id is correct